recipe/meta.yaml

```diff
@@ -8,6 +8,8 @@ package:
 source:
   url: https://github.com/andreasvc/pyre2/archive/v{{ version }}.tar.gz
   sha256: 287f75e1612d4540625c9c6fea88adb26bcd4e1e83cf74e8bbef1b019a57f332
+  patches:
+    - setup-py.patch
 
 build:
   number: 1
```

Comment:
```suggestion
  number: 2
```

Reply:
Yup, that was in the previous one and got lost in the reshuffle (fixed)